realm/realm-library/src/main/java/io/realm/RealmFieldType.java

```diff
@@ -45,15 +48,29 @@
     DOUBLE(10),
     OBJECT(12),
     LIST(13),
-    LINKING_OBJECTS(14);
+    LINKING_OBJECTS(14),
+
+    @SuppressWarnings("PointlessArithmeticExpression")
+    INTEGER_LIST(LIST_OFFSET + 0),
```

Comment:
We do really need all of these? If yes, you are missing `BYTE_LIST` and `BLOB_LIST`

Reply:
I confirmed that core supported list of BLOB.
I've added `RealmFieldType.BINARY_LIST`.